src/shortcuts/__tests__/toggleSort.ts

```diff
@@ -195,6 +195,91 @@ describe('DOM', () => {
     expect(thoughtChildren.map((child: HTMLElement) => child.textContent)).toMatchObject(['1', '2', '3'])
   })
 
+  describe('sort with Global Sort settings', () => {
+    it('thoughts are sorted alphabetically when "Global Sort" settings is Alphabetical', async () => {
+      const thoughtValue = 'c'
+      store.dispatch([
+        newThought({ value: thoughtValue }),
```

Comment:
It would be preferable to hardcode `c` rather than using `thoughtValue`.

Unlike in normal programming, in tests you want to hardcode duplicate literal values in order to maximize concreteness. This reduces the chance of a false positive.

This doesn't extend to complex values, e.g. `await findThoughtByText('c')`, as duplicating the full expression decreases readability and is more likely to introduce a bug than using a variable.

Reply:
> This doesn't extend to complex values, e.g. await findThoughtByText('c'), as duplicating the full expression decreases readability and is more likely to introduce a bug than using a variable.

Sorry, I didn't understand this part.

Reply:
I just mean that hardcoding should only be done for literal values (strings, numbers) and not longer expressions.